monai/transforms/spatial/dictionary.py

```diff
@@ -665,6 +665,32 @@ def __call__(self, data):
             d[key] = zoomer(d[key])
         return d
 
+class TestIdentityd(MapTransform):
+    """
+    This is a pass through transform to be used for testing purposes.  It allows
```

Comment:
2 blank spaces in "... testing purposes. It allows ..."
Suggest just saying a wrapper like other transforms:
**dictionary-based wrapper of :py:class:`monai.transforms.utility.array.TestIdentity`.**